assets/wizards/engagement/views/reader-activation/campaign.js

```diff
@@ -40,6 +46,44 @@ export default withWizardScreen( () => {
 			.finally( () => setInFlight( false ) );
 	};
 
+	/**
+	 * Display prompt requiring editors to confirm skipping, on confirmation send request to
+	 * server to store skipped option in options table and redirect back to RAS
+	 *
+	 * @return {void}
+	 */
+	async function onSkipCampaignSetup() {
+		if (
+			! utils.confirmAction(
+				__(
+					'Are you sure you want to skip setting up a reader activation campaign?',
+					'newspack-plugin'
+				)
+			)
+		) {
+			return;
+		}
+		setError( false );
+		setSkipped( { ...skipped, status: 'pending' } );
+		try {
+			const request = await apiFetch( {
+				path: '/newspack/v1/wizard/newspack-engagement-wizard/reader-activation/skip-campaign-setup',
+				method: 'POST',
+				data: { skip: ! skipped.isSkipped },
+			} );
+			if ( ! request.updated ) {
+				setError( { message: __( 'Server not updated', 'newspack-plugin' ) } );
+				setSkipped( { isSkipped: false, status: '' } );
+				return;
+			}
+			setSkipped( { isSkipped: Boolean( request.skipped ), status: '' } );
+			newspack_engagement_wizard.is_skipped_campaign_setup = request.skipped ? '1' : '';
```

Comment:
After this could we automatically transition to the next view? I clicked the "Skip" button and didn't realize I still had to click "Continue" to proceed.